ZelBack/src/services/appsService.js

```diff
@@ -7837,9 +7855,8 @@ async function updateAppGlobalyApi(req, res) {
         hash: messageHASH,
         timestamp,
         signature,
+        arcaneSender: isEnterpriseApp,
```

Comment:
I think we should put some special key into bench/sas and arcane node should be signing the message with the key and attach signature. 
As this approach can be still spoofed by standard nodes - the receiving node must pretty much verify that the sender is indeed arcane and that can only be done if the sender is signing a message signed by bench/sas

Reply:
Edited...